editors/emacs/Portfile

```diff
@@ -52,18 +52,10 @@ if {$subport eq $name || $subport eq "emacs-app"} {
 }
 
 if {$subport eq "emacs-devel" || $subport eq "emacs-app-devel"} {
-    version         20170918
-    revision        1
+    version         HEAD
```

Comment:
IIRC, we don't use HEAD as version.

Reply:
what do you use when a number doesn't make sense?